tests/outlineCompiler_test.py

```diff
@@ -596,6 +608,62 @@ def test_optimized_default_but_no_nominal_widths(self, FontClass):
         assert private.defaultWidthX == 500
         assert private.nominalWidthX == 0
 
+    def test_underline(self, testufo):
+        # Test with no lib key
+        compiler = OutlineOTFCompiler(testufo)
+        compiler.compile()
+
+        post = compiler.otf["post"].underlinePosition
+
+        cff = compiler.otf["CFF "].cff
+        cff_underline = cff[list(cff.keys())[0]].UnderlinePosition
+
+        assert post == -200
+        assert cff_underline == -200
+
+        # Test with a lib key and postscriptUnderlinePosition
```

Comment:
these could be separate unit-test methods with decriptive names, or even parametrized with/without lib key and info attr and expected, etc.

Reply:
Did separate methods vs parametrized, was quicker.